scripts/api_generator.py

```diff
@@ -2399,11 +2628,14 @@ def __init__(self, name: str, public: Optional[bool], prefix: str,
         self.prefix = prefix
         self.enums = enums
         self.classes = classes
+        self.flags = flags
```

Comment:
Why are these separate now? They're still things in the namespace, right? And it seems like we do the same thing for them as for classes. If it's only for typing, I think it's better to combine them in a `members: List[Union[Class, Flags]]`. Actually, maybe enums should also be in that list, unless they need to be treated specially.

Reply:
Check, I've introduced a base class for flags, enums and classes.

Reply:
Oh, that's even nicer than a union, thanks!